### PR TITLE
Blog app - controllers

### PR DESCRIPTION
**Pull request by Feliverse**
**Project requirements**

- Add RSpec as a gem to your project.
- Create a Request spec file for all your controllers.
- Make sure that for each action you check:
   - If response status was correct.
   - If a correct template was rendered.
   - If the response body includes correct placeholder text.

### CODE REVIEW

File: spec/controllers/posts_spec.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+    describe "GET /users/id/posts/id" do
+      it "returns http success" do
+        get "/users/:user_id/posts/:post_id/"
+        expect(response).to have_http_status(:success)
+        expect(response.body).to include('show post')
+      end
+    end
+  end
+end
```
Comment (cynthiainga) [inline spec/controllers/posts_spec.rb:19]: - [x] Good job so far, but you should test if a correct template was rendered for `show` and `index` actions like `expect(response).to render_template(:show)`. Please kindly add them 🙃 🙃

File: spec/controllers/users_spec.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  end
+
+  describe "GET /show" do
+    it "returns http success" do
+      get "/users/show"
+      expect(response).to have_http_status(:success)
+      expect(response.body).to include('show user')
+    end
+  end
+end
```
Comment (cynthiainga) [inline spec/controllers/users_spec.rb:19]: - [x] Good job so far, but you should test if a correct template was rendered for `show` and `index` actions like `expect(response).to render_template(:show)`. Please kindly add them 🙃 🙃

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @Feliverse, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [x] Has and uses RSpec gem for testing ✔️ 
- [x] Has `users_spec.rb` and `posts_spec.rb` ✔️ 
- [x] No linters errors ✔️ 

## Required Changes ♻️

- [x] Please kindly check the comments under the review 👇🏾 👇🏾 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/controllers/posts_spec.rb (hunk 1 above)
Comment (Feliverse) [reply spec/controllers/posts_spec.rb:19]: Oh, I will add this specs in a while, thank you

File: spec/controllers/users_spec.rb (hunk 2 above)
Comment (Feliverse) [reply spec/controllers/users_spec.rb:19]: Oh, I will add this specs in a while, thank you

**Review by anagudelogu — APPROVED** [review]
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi @Feliverse, 
I'm Andres.

You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 



## Highlights
✅ Great use of Git flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Descriptive PR title, summary and commit messages. 💬 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**